web/src/app/pages/create-survey/name-job/name-job.component.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Component, Input, OnChanges} from '@angular/core';
+import {FormGroup, FormBuilder} from '@angular/forms';
+
+@Component({
+  selector: 'name-job',
+  templateUrl: './name-job.component.html',
```

Comment:
This name feels a little clunky to me, and it seems like it restricts it pretty narrowly to the job name. Maybe something like job-details-form instead?

Reply:
Sg, renamed to `job-details` to keep it short.